Services/OpenIdConnect/classes/class.ilAuthProviderOpenIdConnect.php

```diff
@@ -108,7 +105,7 @@ public function doAuthentication(\ilAuthStatus $status)
             $_GET['target'] = (string) $this->getCredentials()->getRedirectionTarget();
 
             if ($this->settings->getLogoutScope() == ilOpenIdConnectSettings::LOGOUT_SCOPE_GLOBAL) {
-                $token = $oidc->requestClientCredentialsToken();
+                $token = $oidc->requestUserInfo();
                 ilSession::set('oidc_auth_token', $token->access_token);
```

Comment:
We are still storing the "Access Token" here. In the changed code above, you renamed the variable to `$id_token`, 
 and "ID Token" != "Access  Token". We should use the correct wording.

Reply:
Sorry,  I messed up c&p from release_8 branch.